src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -55,12 +59,12 @@ const CourseEnrollments = ({ children }) => {
     const assignmentsData = sortAssignmentsByAssignmentStatus(data);
     setAssignments(assignmentsData);
 
-    const hasCancelledAssignments = assignmentsData?.some(
-      assignment => assignment.state === ASSIGNMENT_TYPES.CANCELLED,
-    );
-    const hasExpiredAssignments = assignmentsData?.some(assignment => isAssignmentExpired(assignment));
+    const hasActiveCancelledAssignments = assignmentsData?.some((assignment) => (
+      assignment.state === ASSIGNMENT_TYPES.CANCELLED)) && getIsActiveCancelledAssignment(assignmentsData);
```

Comment:
`ASSIGNMENT_TYPES.CANCELLED_NOTIFICAITON` here, too (or change the names of the constant to just `CANCELLED`).

Reply:
I think we are checking for `LearnerContentAssignment` states instead of `AssignmentActions` so `ASSIGNMENT_TYPES.CANCELLED` should be correct. please let me know if i'm missing something!

Reply:
ah, sorry, you are correct! Need more coffee!